source/JsMaterialX/test/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "pretest": "node ./node_modules/copy/bin/cli.js ../_build/source/JsMaterialX/MaterialXLib.* ./_build/",
+    "test": "mocha '*.spec.js' --require @babel/register",
```

Comment:
There is a typo here. The extra "," at the end should be removed. This was causing the install error mentioned.

Reply:
I removed this typo in the latest commit.